isyfact-standards-doc/src/docs/antora/modules/isy-security/pages/konzept/inhalt.adoc

```diff
@@ -121,17 +121,18 @@ Der `RolePrivilegeGrantedAuthoritiesConverter` ist für die Konvertierung der vo
 
 Die Absicherung der Service-Schnittstellen von IsyFact-Anwendungen erfolgt über folgende AOP-Annotationen:
 
-- Der `NutzerAuthentifizierunginterceptor` ist für die Authentifizierung von technischen Nutzern zuständig und wird über eine Annotation `@NutzerAuthentifikation` bereitgestellt.
-Parallel dazu existiert die Annotation `@ClientAuthentification` zur Authentifizierung von OAuth2.0 Clients.
-Dies ermöglicht die Umstellung von technischen Nutzern auf OAuth2.0 Clients, ohne große Auswirkungen im Anwendungssystem zu erzeugen.
-Die Authentifizierungsdaten werden aus der Konfiguration der Anwendung ausgelesen und zur Authentifizierung verwendet.
-Die Authentifizierung findet über die Implementierung des Interfaces `Authentifizierungsmanager` statt.
-
-- Mit der von Spring Security bereitgestellten Annotation `@secured` werden angebotene Services der Anwendung abgesichert.
+- Mit der von Spring Security bereitgestellten Annotation `@Secured` werden angebotene Services der Anwendung abgesichert.
 Dieser Annotation muss noch der Name eines anwendungsspezifischen Rechts mitgegeben werden, das für die Verwendung dieses Service benötigt wird.
 Im Interceptor `SecuredInterceptor` wird sichergestellt, dass nur dann der Zugriff gewährt wird, wenn der angemeldete Nutzer über seine Rollen auch dieses anwendungsspezifische Recht besitzt.
 
-Hinweis: Die Annotation `@secured` löst die Annotation `@gesichert` ab und ist analog dazu verwenden.
+Hinweis: Die Annotation `@Secured` löst die Annotation `@Gesichert` ab und ist analog dazu verwenden.
+
+Eine Absicherung von Services auf Ebene des Ressourcenbesitzers (Resource Owner), also eines technischen Nutzers bzw. OAuth-2-Client bietet die IsyFact-Annotation `@Authenticate`.
+Dieser Annotation muss noch die oauth2ClientRegistrationId mitgegeben werden, unter der der Ressourcenbesitzer in der betrieblichen Konfiguration registriert ist.
+Im Interceptor `AuthenticateInterceptor` erfolgt eine Authentifizierung über die Methode `authentifiziere(oauth2ClientRegistrationId)` im Authentifizierungsmanager.
+Der Service wird dann im Namen und mit den Rechten des Ressourcenbesitzers ausgeführt.
+
+Hinweis: Die Annotation `@Authenticate` löst die Annotation `@NutzerAuthentifikation` ab und erweitert sie um die Funktion, auch hinterlegte OAuth-2-Clients authentifizieren zu können.
```

Comment:
Werden nicht-Oauth2.0-Clients, also die "alten" techn. Nutzer, auch darüber authentifiziert?

Reply:
Da die Annotation als Parameter ‚oauth2ClientRegistrationId‘ hat und intern die Methode authentifiziere(oauth2ClientRegistrationId)verwendet wird, werden die selben Arten von Benutzern auf die selbe Weise authentifiziert: über die oauth2ClientRegistrationId werden die in der Konfig hinterlegten Authentifizierungsdaten ausgelesen und je nach Typ (techn.Nutzer = Resource Owner bzw. Oauth2.0-Client = Client Cred.) der entsprechende Authentifizierungsflow gewählt. 

Das dachte ich in der Beschreibung des AuthenticateInterceptors beschrieben zu haben und auf die Annotation bezogen zu haben. 
Mit ‚erweitert‘ und ‚auch‘ wollte ich ausdrücken, dass @Authenticate zusätzlich zu der bisher mit @NutzerAuthentifikation möglichen Authentifizierung von nicht-Oauth2.0-Clients, also die "alten" techn. Nutzer auch Oauth2.0-Clients authentifiziert werden können – bzw. Benutzer, die in der Konfig unter der oauth2ClientRegistrationId abgelegt sind, mit dem zu verwendenden Auth-Typ.